modules/setting/mirror.go

```diff
@@ -32,7 +32,7 @@ func newMirror() {
 	// - please note this was badly named and only disabled the creation of new pull mirrors
 	if Cfg.Section("repository").Key("DISABLE_MIRRORS").MustBool(false) {
 		log.Warn("Deprecated DISABLE_MIRRORS config is used, please change your config and use the options within the [mirror] section")
-		// TODO: enable on v1.17.0: log.Error("Deprecated fallback used, will be removed in v1.18.0")
+		log.Error("Deprecated fallback used, will be removed in v1.18.0")
```

Comment:
I recommend we switch the Warn to Error and adjust the error. Then add a `// FIXME: Deprecation notice to remove in v1.18`

Reply:
Actually see my patch